pkg/dnservice/cfg.go

```diff
@@ -65,6 +72,15 @@ type Config struct {
 	// RPC configuration
 	RPC rpc.Config `toml:"rpc"`
 
+	Ckp struct {
+		ScannerInterval    int64 `toml:"scanner-interval"`
+		ExecutionInterval  int64 `toml:"execution-interval"`
+		FlushInterval      int64 `toml:"flush-interval"`
+		ExecutionLevels    int16 `toml:"execution-levels"`
+		CatalogCkpInterval int64 `toml:"catalog-ckp-interval"`
+		CatalogUnCkpLimit  int64 `toml:"catalog-unckp-limit"`
```

Comment:
use util/toml.Duration type to define these fields. And can use "1s|1ms|1s" to write  config file as below:
scanner-interval = "1s"
execution-interval = "1ms"
...